src/test/java/org/opensearch/ad/ml/CheckpointDaoTests.java

```diff
@@ -1000,4 +1005,77 @@ private double[] getPoint(int dimensions, Random random) {
         }
         return point;
     }
+
+    public void testDeserializeRCFModelPreINIT() throws Exception {
```

Comment:
are the following 2 tests used for single-stream detector's checkpoints? If so, could you add comments?